op-e2e/config/init.go

```diff
@@ -329,7 +329,7 @@ func initAllocType(root string, allocType AllocType) {
 			mtx.Unlock()
 
 			// This needs to be updated whenever the latest hardfork is changed.
-			if mode == genesis.L2AllocsGranite {
+			if mode == genesis.L2AllocsHolocene {
```

Comment:
This change causes `TestPendingGasLimit` to fail. It is also not necessary for the test added on this PR. But it feels like the right change to make...

Reply:
Pinched this off onto https://github.com/ethereum-optimism/optimism/pull/14693